src/swarm-tui/Swarm/TUI/Controller/EventHandlers/Main.hs

```diff
@@ -149,7 +150,7 @@ isRunning = do
 whenRunning :: EventM Name AppState () -> EventM Name AppState ()
 whenRunning a = isRunning >>= \r -> when r a
 
-whenCheating :: EventM Name AppState () -> EventM Name AppState ()
-whenCheating a = do
-  s <- get
-  when (s ^. uiState . uiCheatMode) a
+whenDebug :: DebugOption -> EventM Name AppState () -> EventM Name AppState ()
+whenDebug d a = do
+  debug <- use $ uiState . uiDebugOptions . icontains d
```

Comment:
Why is `icontains` necessary?  Wouldn't `contains` work just as well?

Reply:
It would, I somehow saw `icontains` first and did not question the naming. 😅

Reply:
Hah, it did spare me ambiguity caused by `Swarm.Game.Entity.contains`.